app/controllers/listings_controller.rb

```diff
@@ -32,31 +32,15 @@ class ListingDeleted < StandardError; end
   def index
     @selected_tribe_navi_tab = "home"
 
-    # -- HOTFIX, REMOVE
-    if request.xhr? && params[:person_id] # AJAX request to load on person's listings for profile view
-      @person = Person.find(params[:person_id])
-      PersonViewUtils.ensure_person_belongs_to_community!(@person, @current_community)
-
-      # Returns the listings for one person formatted for profile page view
-      per_page = params[:per_page] || 200 # the point is to show all here by default
-      page = params[:page] || 1
-      render :partial => "listings/profile_listings", :locals => {:person => @person, :limit => per_page}
-      return
-    end
-    # -- HOTFIX, REMOVE
-
     respond_to do |format|
-      format.js do
```

Comment:
Yes, format.js is not the same. It responds to the corresponding HTTP_ACCEPT header-field. Moreover, the order is important as if the HTTP_ACCEPT field contains '*/*', it'll pick the first one.